guides/tutorials/codelabs/pubsub-to-cf/index.md

```diff
@@ -102,31 +104,29 @@ e. Apply your changes:
 
 ## 5. Configure the Application and Pipeline
 
-a. Create a new pipeline for your application. In the pipeline configuration, configure the Expected Artifacts as shown below, a GCS object with the path to the application archive in your GCS bucket:
+a. Create a new pipeline for your application. In the pipeline configuration, under Automated Triggers, add a new trigger of **Type** "Pub/Sub". Select GCP Cloud Pub/Sub as the **Pub/Sub System Type**, select your GCP Cloud Pub/Sub Spinnaker subscription in **Subscription Name**, and under **Attribute Constraints**, add an entry with the key `eventType` and value `OBJECT_FINALIZE ` (see the [Google Cloud Storage documentation](https://cloud.google.com/storage/docs/pubsub-notifications)).
```

Comment:
Can you break this out into a list, or even a table?

Reply:
Good suggestion--thanks! Made a list here (and also for the artifact configuration).